app/assets/javascripts/oxalis/api/api_latest.js

```diff
@@ -670,7 +670,14 @@ class DataApi {
     if (layerName !== segmentationLayerName) {
       throw new Error(messages["mapping.unsupported_layer"]);
     }
-    Store.dispatch(setMappingAction(_.clone(mapping), options.colors, options.hideUnmappedIds));
+    Store.dispatch(
+      setMappingAction(
+        "<code provided>",
```

Comment:
Maybe change to `<user provided mapping>` or `<custom mapping>` - I did not understand what was meant here at first.